_source/user-guide/cloud-siem/drilldown-links.md

```diff
@@ -7,6 +7,7 @@ flags:
 tags:
   - logz.io-security
   - siem
+  - drilldown
```

Comment:
Need to add a slug for new tag for Logz.io drilldowns - and possibly explanation of how these are different from the already documented Kibana drilldowns.